function_calling/notebooks/function_calling_guide.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
I had to add further environment variables with `SAMBASTUDIO_` prefix.
We need to take care of this aliasing.

Reply:
Fixed in Readme